samples/tfx-oss/TFX Example.ipynb

```diff
@@ -19,6 +19,14 @@
     "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.10/kfp.tar.gz --upgrade\n"
    ]
   },
```

Comment:
Can you also add a note that this notebook should be run in the KFP cluster?

Reply:
Done.